### ISSUE TITLE
CheatEngine72.exe Download from releases page is gone

### ISSUE THREAD

**Issue by frostworx** [cheat-engine/cheat-engine#1523]
As the exe was removed from the releases page automated downloads are.
When searching for the last md5sum `8ce5de1b284bc9f865865902fea09489` this seems to affect several tools
including [mine](https://github.com/frostworx/steamtinkerlaunch).
Using download urls directly from github is pretty convenient and easy to maintain, so please consider readding it.

**Comment by frostworx** [comment]
https://chocolatey.org/packages/cheatengine

**Comment by Symbai** [comment]
Why don't you use the direct link download from the ce homepage? `https://cheatengine.org/download/CheatEngine7.2_MissingSetup.rar`

**Comment by frostworx** [comment]
Seems to be the best (only) solution, which I might use as a last resort, if the downloads don't come back on the release page.
I guess you have seen [this](https://github.com/frostworx/steamtinkerlaunch/issues/160#issuecomment-736897776) as the rar url is not really that much advertised, that it would be common sense that it even exists.

**Comment by wefalltomorrow** [comment]
Yup, would be nice for it to be added back. I'll be sticking with 7.1 in the meantime.

**Comment by frostworx** [comment]
I consider uploading the original `8ce5de1b284bc9f865865902fea09489 CheatEngine72.exe` on github, as it would be simply the least work and should be perfectly fine legally, shouldn't it?

**Comment by avboy1337** [comment]
choco install cheatengine

**Comment by frostworx** [comment]
Yes exactly! As already mentioned [here](https://github.com/cheat-engine/cheat-engine/issues/1523#issuecomment-804383448) chocolatey is [also](https://community.chocolatey.org/packages/cheatengine#files) [affected](https://community.chocolatey.org/packages/cheatengine#comment-5311484214).

**Comment by RokkumanX** [comment]
> I consider uploading the original 8ce5de1b284bc9f865865902fea09489 CheatEngine72.exe on github, as it would be simply the least work and should be perfectly fine legally, shouldn't it?

Why would it be illegal? If the Cheat Engine team pulled our legs so could we, I say screw em all!

[CheatEngine72.zip](https://github.com/cheat-engine/cheat-engine/files/6684493/CheatEngine72.zip) <--- Last clean 7.2 release from GitHub downloaded February 1st 2021

Hope it helps someone!

**Comment by FerdieMan** [comment]
This file contains a virus and both chrome and edge will block the download.  ( `Trojan:Script/Wacatac.B!ml` )

**Comment by frostworx** [comment]
The md5sum of the CheatEngine72.exe in RokkumanX's zip package is identical to the last official one (`8ce5de1b284bc9f865865902fea09489`), so I'd assume it is a false positive triggered by zipping the package.
Either way good to be careful and thanks for both the warning and thanks to both for trying to help others.


_(I think I will upload the original, unzipped exe as well on github, when I continue working on my program)_

**Comment by RokkumanX** [comment]
Well, all I really did was to right click the CheatEngine72.exe and chose **Send to -> Compressed (zipped) folder** within Windows 10 itself, there should be no virus or anything and Windows Defender didn't find anything when I scanned both the .zip nor the .exe

The MD5 checksum matches all known CheatEngine72.exe (24,2 MB (25 382 440 bytes)) versions downloaded from the Cheat Engine release page before it went away.

**Comment by frostworx** [comment]
Thanks for the explanation, didn't know that exe extension isn't supported.
I fully trust your upload. Thanks again for your friendly help!

**Comment by SaifRushdHadad** [comment]
You can always just utilize the wayback machine archive.
https://web.archive.org/web/20210118065254if_/https://github.com/cheat-engine/cheat-engine/releases/download/7.2/CheatEngine72.exe

**Comment by Icekiss69** [comment]
> > I consider uploading the original 8ce5de1b284bc9f865865902fea09489 CheatEngine72.exe on github, as it would be simply the least work and should be perfectly fine legally, shouldn't it?
> 
> Why would it be illegal? If the Cheat Engine team pulled our legs so could we, I say screw em all!
> 
> [CheatEngine72.zip](https://github.com/cheat-engine/cheat-engine/files/6684493/CheatEngine72.zip) <--- Last clean 7.2 release from GitHub downloaded February 1st 2021
> 
> Hope it helps someone!



> You can always just utilize the wayback machine archive. https://web.archive.org/web/20210118065254if_/https://github.com/cheat-engine/cheat-engine/releases/download/7.2/CheatEngine72.exe

Hi
do you have a working download link for cheat engine on github version 7.1 / 7.0 and older? and some 6.0 versions?
